service/controller/app.go

```diff
@@ -0,0 +1,122 @@
+package controller
+
+import (
+	"github.com/giantswarm/apiextensions/pkg/apis/application/v1alpha1"
+	"github.com/giantswarm/apiextensions/pkg/clientset/versioned"
+	"github.com/giantswarm/app-operator/service/controller/v1"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"github.com/giantswarm/operatorkit/client/k8scrdclient"
+	"github.com/giantswarm/operatorkit/controller"
+	"github.com/giantswarm/operatorkit/informer"
+	apiextensionsclient "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
+	"k8s.io/client-go/kubernetes"
+	//"github.com/giantswarm/app-operator/service/controller/v1"
+)
+
+const AppControllerSuffix = "-app"
```

Comment:
I don't think this is needed. Only chart-operator seems to have it. Lets remove it and just use the project name as the controller name.